### ISSUE TITLE
Connect Movie DB API

### ISSUE THREAD

**Issue by truongnguyenlinh** [truongnguyenlinh/flixlist#10]
https://developers.themoviedb.org/3/getting-started/introduction

**Comment by Hunajpu** [comment]
I think I'm done with this issue

**Comment by truongnguyenlinh** [comment]
Is there a PR for this?

**Comment by truongnguyenlinh** [comment]
Linked your PR, please close only once your PR is merged into main.
